tests/scripts/thread-cert/Cert_5_6_05_NetworkDataRegisterAfterAttachRouter.py

```diff
@@ -133,6 +120,54 @@ def test(self):
             if addr[0:3] == '200':
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        MED = pv.vars['MED']
+        SED = pv.vars['SED']
+        _lpkts = pkts.filter_wpan_src64(LEADER)
+
+        # Step 1: Ensure the topology is formed correctly
+        _lpkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).filter_wpan_dst64(SED).must_next()
+
+        # Step 3: The DUT Automatically sends a CoAP Response frame and
+        # MLE Data Response message
+        _lpkts.copy().filter_coap_ack(SVR_DATA_URI).must_next()
+
+        _lpkts_med = _lpkts.copy()
+        _lpkts_sed = _lpkts.copy()
+
+        # Step 4: The DUT MUST send a multicast MLE Data Response with
```

Comment:
Are we checking that the MLE Data Response is sent via multicast?

Reply:
added multicast dst filter